ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/cesium/geocoder.js

```diff
@@ -38,66 +37,72 @@ module.exports = Backbone.Model.extend({
   },
   queryGazetteerOnline(input) {
     // Load in request string
-    return Cesium.loadJson(geocoderOnlineEndpoint + input)
-      .then(function(results) {
-        var bboxDegrees
-
-        // Extract desired fields
-        return results.map(function(resultObject) {
-          bboxDegrees = resultObject.boundingbox
-          return {
-            displayName: resultObject.display_name,
-            destination: Cesium.Rectangle.fromDegrees(
-              bboxDegrees[2],
-              bboxDegrees[0],
-              bboxDegrees[3],
-              bboxDegrees[1]
-            ),
-          }
-        })
-      })
-      .otherwise(
-        function(error) {
-          onlineGazetteer = false
-          // Run query against offline gazetteer
-          return this.queryGazetteerOffline(geocoderOfflineEndpoint, input)
-        }.bind(this)
-      )
-  },
-  queryGazetteerOffline(input) {
-    // Load in request string
-    return Cesium.loadText(geocoderOfflineEndpoint + input)
-      .then(function(results) {
-        // Strip out unwanted characters and parse JSON
-        var jsonResult = /\((.+)\)/.exec(results)[1]
-        var jsonObject = JSON.parse('[' + jsonResult + ']')
-
-        // Extract desired fields
-        return jsonObject.map(function(locationResult) {
-          var resultValues = locationResult.resourceSets[0].resources[0]
-          if (typeof resultValues != 'undefined') {
+    return new Promise((resolve, reject) => {
+      $.ajax({
+        url: geocoderOnlineEndpoint + input,
+        success: function(results) {
+          const formattedResults = results.map(function(resultObject) {
+            const bboxDegrees = resultObject.boundingbox
             return {
-              displayName: resultValues.name,
+              displayName: resultObject.display_name,
               destination: Cesium.Rectangle.fromDegrees(
-                resultValues.bbox[1],
```

Comment:
❓ Is the Bounding Box coordinate order different for the online vs offline gazetteer?

Reply:
This is how it has always been. See master: 
https://github.com/codice/ddf/blob/master/ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/cesium/geocoder.js#L51-L85